src/vivarium/framework/values/manager.py

```diff
@@ -676,3 +736,8 @@ def get_value(self, name: str) -> Pipeline:
 
         """
         return self._manager.get_value(name)
+
+    # TODO: [MIC-5452] Remove this method (attributes should be obtained via population views)
+    def get_attribute(self, name: str) -> AttributePipeline:
```

Comment:
NOTE: this will NOT stick around for the long haul. The way we intend for users to get attributes is via the population_view.get() method. This is just temporarily needed in a few loations (pytests and examples) until that interface gets updated.